doc/todo/RELEASE

```diff
@@ -25,265 +12,78 @@ Following shell expansions are used in this document:
- strace -o ~elektra/$VERSION/umount.strace kdb umount user:/release_test
-
-Run tests:
-
- KDB=kdb kdb run_all -v | tee ~elektra/$VERSION/run_all 2>&1
- KDB=kdb-full kdb-full run_all | tee ~elektra/$VERSION/run_all_full 2>&1
-
-Install libelektra-test of the previous version and run again:
-
- CHECK_VERSION=NO kdb run_all -v > ~elektra/$VERSION/run_all_abi 2>&1
-
-
-== Test external tools ==
-
-
-Check if tools based on Elektra still do:
- dq
-
-Run unit-tests in-source again (because some use installed Elektra):
- make run_all
-
-
-== Test external test suites ==
-
-#https://github.com/oyranos-cms/oyranos/blob/master/src/tests/test2.cpp#L185
-#https://github.com/oyranos-cms/oyranos/blob/master/src/liboyranos_config/oyranos_elektra.c#L195
-
-git clone git@github.com:oyranos-cms/oyranos.git
-cd oyranos
-mkdir build
-cd build
-cmake .. # might fail, simply comment out missing libs
-make -j5 # not needed
-make test2 # or make test-2 for the most of the tests
-src/tests/test2 Elektra
-
-
-== Documentation ==
-
-And the API docu:
- mkdir ~elektra-doc/api/$VERSION/
- cd ~build && cmake -DBUILD_PDF=ON . && rm -rf doc && make html man && cp -ra doc/html doc/latex doc/man ~elektra-doc/api/$VERSION/
-
-Symlink current to latest version and add everything:
- cd ~elektra-doc/api/ && rm current && ln -s $VERSION current && git add current $VERSION && git commit -a -m "$VERSION Release"
-
-
-
+- https://build.libelektra.org/job/libelektra/job/master/ must pass
+
+= When Source Code is considered ready =
+
+- Trigger release pipeline with parameters
+  On a release of a new version, the revision parameters should be set to 1 (default)
+- Download artifacts after release build is finished and pipeline waits for input:
+  To download the artifacts either open the "Artifacts" tab on Blue Ocean or open this URL
+  https://build.libelektra.org/job/libelektra-release/<BUILD_NUMBER>/artifact/artifacts/
+  where <BUILD_NUMBER> is the number of the Jenkins build e.g. 77
+- Inspect the logs from the artifacts
+  For each distribution two artifact folder exist: "<DISTRO_CODENAME>" and "<DISTRO_CODENAME>-installed"
+  "<DISTRO_CODENAME>-installed" contains the test and strace logs of Elektra installed through the built packages
+  For <DISTRO_CODENAME> artifact:
+    - [ ] Error logs (*.error) must be empty
+    - [ ] Check in the (test-*) folders if the passed tests really were successful
+    - [ ] Check if version is correct in the version file
+    - [ ] Check if the package build log has warnings (<DISTRO_CODENAME>/elektra_*.build and <DISTRO_CODENAME>/elektra_*.error)
+    - [ ] Check if all plugins/bindings/tools that are part of a package were included (<DISTRO_CODENAME>/elektra_*.build)
+  For <DISTRO_CODENAME>-installed artifact:
+    - [ ] Check in the (test-*) folders if the passed tests really were successful
+- Inspect the changes to the libelektra git repository in git/master.log
+  - [ ] Check diff of commit "Regenerate dot of plugins for release" and if
```

Comment:
```suggestion
  - [ ] Check diff of commit "release: regenerate plugins overview picture" and if
```
We usually use "topic: <short msg>" for git commit messages. Can you add release: for all the messages done during the release?